web/src/components/feature/results/PhaseTable.tsx

```diff
@@ -34,7 +34,7 @@ export const PhaseTable = (props: {
         'Phases',
         'Ratio',
         'Mass Concentration',
-        'Mole Concentration',
+        'Mole Concentration (mol)',
```

Comment:
Not sure if the labels were correct. I think they should reflect that the numbers are either the concentration of mercury measured volumetric or by mole. "Mercury concentration" fits for both, but is bad since it does not separate enough. "Mercury Mass/Mole concentration" is very describing, but perhaps too long. Not sure what to go for here ...

Reply:
I think "Mercury Mass/Mole concentration" is too long. But not sure what the best header for this column is. Can remove mol, and try to put units in all colums, to see if that looks better.

Reply:
I suggest;
- Phases
- Fractions
- Concentration (mol)
- Concentration (ug)

Reply:
Stig Oskar's suggestion:
![image](https://user-images.githubusercontent.com/47739370/198216931-a705eac3-e107-4575-9122-9667a932df6e.png)

or with units in all columns: 
![image](https://user-images.githubusercontent.com/47739370/198217706-5f39db7d-bc04-4f0a-ba7a-7adcb57caf6c.png)

Reply:
Oh, and I think the k in kg should be lower case.

Reply:
![image](https://user-images.githubusercontent.com/47739370/198222330-fff8c25a-ebf1-46ec-9178-d8ed667e974a.png)

Reply:
I'm fine with this. 💁‍♂️ Can always be changed later on.

Reply:
Could leave it like this for now, and than change later if we find a better solution